### ISSUE TITLE
unable to load preload script. cannot read property of undefined.

### ISSUE THREAD

**Issue by cnshu** [timlg07/custom-electron-titlebar-minimal-example#3]
I used tis minimal example but it thrown me an error what should i do 
![Screenshot (63)](https://user-images.githubusercontent.com/83962514/135127076-15a09f82-48f7-45b5-99a0-917a64915191.png)

**Comment by LonelyInstantN** [comment]
exact same problem on me 
"custom-electron-titlebar": "^3.2.7",
"electron": "^15.0.0"
"vue-cli-plugin-electron-builder": "~2.1.1"
"vue": "^3.0.0"
I have no clue 
let me know if you fixed

**Comment by cnshu** [comment]
I solved using the electron version 12.0.0. This version is completely stabke

**Comment by cnshu** [comment]
I solved using the electron version 12.0.0. This version is completely adaptable with electron-custom-builder 

``` "dependencies": {
        "custom-electron-titlebar": "^3.2.7",
        "internet-available": "^1.0.0",
        "mysql": "^2.18.1"
    },
    "devDependencies": {
        "electron": "^12.0.0",
        "electron-builder": "^22.11.7"

    }```

**Comment by cnshu** [comment]
I solved my problem using the older electron version i.e version 12.0.0 which works finely with the code provided by this guy.

```"dependencies": {
        "custom-electron-titlebar": "^3.2.7",
        "internet-available": "^1.0.0",
        "mysql": "^2.18.1"
    },
    "devDependencies": {
        "electron": "^12.0.0",
        "electron-builder": "^22.11.7"

    }```

**Comment by cnshu** [comment]
I solved my problem using the older electron version i.e version 12.0.0 which works finely with the code provided by this guy.

`
dependencies: {
        "custom-electron-titlebar": "^3.2.7",
        "internet-available": "^1.0.0",
        "mysql": "^2.18.1"
    },
    "devDependencies": {
        "electron": "^12.0.0",
        "electron-builder": "^22.11.7"

    }

`

**Comment by cnshu** [comment]
I solved my problem using the older electron version i.e version 12.0.0 which works finely with the code provided by this guy.

**"dependencies": {
        "custom-electron-titlebar": "^3.2.7",
        "internet-available": "^1.0.0",
        "mysql": "^2.18.1"
},

 "devDependencies": {
        "electron": "^12.0.0",
        "electron-builder": "^22.11.7" 
 }**
